tests/ctst/world/Zenko.ts

```diff
@@ -101,19 +182,22 @@ export default class Zenko extends World {
      * @param {object} result - contains both stderr and stdout from the CLI command.
      * @returns {boolean} - if the result is a success or a failure
      */
-    checkResult(result: any[]): boolean {
+    checkResult(result: Result | Utils.Command | Result[]): boolean {
+        let usedResult: Result | Utils.Command | Result[];
```

Comment:
```suggestion
    checkResult(result: Result | Result[]): boolean {
        let usedResult: Result | Result[];
```

I do not see any use of `Utils.Command` in this function, as `result` is always casted to `result`, it might be needed to update the caller functions here, it seems strange to mix types...

Reply:
responded below